src/reference/01-General-Info/05-Using-Sonatype.md

```diff
@@ -94,12 +143,15 @@ optional dependencies in our artifact:
 pomIncludeRepository := { _ => false }
 ```
 
-Next, the POM metadata that isn't generated by sbt must be added. This
-is done through the `pomExtra` configuration option:
-
-```scala
+To publish to a maven repository, you'll need to configure a few
+settings so that the correct metadata is generated.
+Specifically, the build should provide data for `organization`, `url`,
+`license`, `scm.url`, `scm.connection` and `developer` keys. To generate a 
```

Comment:
Based on the `build.sbt` for sbt, I believe the following are useful examples of those other keys:

```scala
scmInfo := Some(
  ScmInfo(
    url("https://github.com/your-account/your-project"),
    "scm:git@github.com:your-account/your-project.git"
  )
)

developers := List(
  Developer(
    id    = "Your identifier",
    name  = "Your Name",
    email = "your@email",
    url   = url("http://your.url")
  )
)
```